tests/mail/tests.py

```diff
@@ -1084,12 +1084,11 @@ def test_sanitize_address(self):
                 "<to@example.com>",
             ),
             (
-                ("Tó Example very long" * 4, "to@example.com"),
```

Comment:
Removed duplicate test from line 1079.

Reply:
This is not a duplicate. Please revert this change and add a separate entry to tests.